easybuild/easyconfigs/e/EMAN2/EMAN2-2.21a-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMake'
+
+name = 'EMAN2'
+version = '2.21a'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image processing suite
+ with a primary focus on processing data from transmission electron microscopes. """
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/cryoem/eman2/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['EMAN2-2.21a.patch']
```

Comment:
@smoors Please use `%(version)s` rather than hardcoding `2.21a`, and make the patch filename a bit more descriptive, e.g. `EMAN2-%(version)s_fix-custom-install-prefix.patch`

Reply:
@boegel I will make the change, but I'm a little annoyed by it.
What's the benefit of using `%(version)s` if we have to change it back to the hard-coded version in future versions of EMAN2? I suggest we choose one of the following:
1) use `%(version)s` always and make new patches for every new version
2) use the hard-coded version always and try to re-use patches as much as possible

Reply:
It helps with realizing that the patch may need to be updated too when the version is bumped.

Sometimes you get lucky and you can reuse the patch that was made for a previous version though.